test/a11y.html

```diff
@@ -47,13 +47,11 @@
 
         test('aria attributes', function() {
           assert.equal(collapse.getAttribute('role'), 'group');
-          assert.equal(collapse.getAttribute('aria-expanded'), 'false');
```

Comment:
You can actually assert that it's undefined and add a comment explaining why for future reference.

Reply:
Done.